report/src/conclusion/conclusion.tex

```diff
@@ -0,0 +1,13 @@
+\chapter{Conclusion}
+This report examines a stakeholder's problems regarding the management of meetings. 
+By performing a usability evaluation of a developed prototype, it is concluded that a solution to their problem might involve a presence detection system to establish whether the stakeholder's meeting rooms are occupied. 
+Chapter \ref{chap:presence_intro} describes and evaluates technologies appropriate for presence detection, guided by a set of criteria considered necessary to address the stakeholder's problem. 
+Of these technologies, Bluetooth Low Energy was found to be the best candidate. 
+Three techniques for presence detection using BLE were presented and analyzed in Chapter \ref{sec:presence_detection_using_ble}. 
+It was found that fingerprinting using k nearest-neighbors would be a suitable candidate for presence detection.
+
+Subsequently, we conducted an experimental evaluation to determine if the selected technique could effectively contribute to a system aimed at resolving the stakeholder's problems. 
+The development of a mobile application, implemented in C\#, was created to facilitate these experiments. 
+The results of the experiments indicated that our approach could achieve satisfactory accuracy in detecting whether a person was present within the room where the experiments were carried out.
```

Comment:
```suggestion
After conducting an experimental evaluation to assess the effectiveness of the selected technique in addressing the stakeholder's problems, we proceeded with the development of a mobile application implemented in C#. 
This application served as a facilitator for the experiments conducted. 
The obtained results from these experiments demonstrated that our approach yielded satisfactory accuracy in detecting the presence of individuals within the experimental room.
```

Reply:
I think this needs to be rephrased as well, it reads quite poorly. For example, this sentence:
"The development of a mobile application, implemented in C\#, was created to facilitate these experiments." doesn't make any sense. Removing the interposed sentence leaves us with:
"The development of a mobile application was created to facilitate these experiments.", implying that we created a development. 

Also, when talking about the results of the experiment, should we reference back to that chapter? You did that in several other cases, but why not with the experimental results too?